src/main/java/org/sonatype/repository/conan/internal/proxy/matcher/ConanMatcher.groovy

```diff
@@ -119,6 +121,18 @@ class ConanMatcher
     )
   }
 
+  /**
+   * Matches on ping endpoint
+   */
+  static Builder ping() {
+    new Builder().matcher(
+        and(
+            new ActionMatcher(GET),
+            new TokenMatcher(PING)
```

Comment:
Suggestion: You probably don't need a TokenMatcher here, and might be able to get away with a LiteralMatcher.

Reply:
https://github.com/sonatype-nexus-community/nexus-repository-conan/pull/45/commits/5585a36ecdceec2551ad34bf496df0fe00be97fd